src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -1559,8 +1559,13 @@ private static ParameterSymbol GetParameter(int argIndex, MemberAnalysisResult r
                                                    ref useSiteDiagnostics,
                                                    out okToDowngradeToNeither);
 
-                var type1Normalized = type1.NormalizeTaskTypes(Compilation);
-                var type2Normalized = type2.NormalizeTaskTypes(Compilation);
+                var type1Normalized = type1;
+                var type2Normalized = type2;
+                if (!_binder.InAttributeArgument)
+                {
+                    type1Normalized = type1.NormalizeTaskTypes(Compilation);
```

Comment:
>NormalizeTaskTypes [](start = 44, length = 18)

It looks like there are two more call sites for this method in this file. Should they be protected as well?

Reply:
Possibly. However I can't figure out what conditions cause us to reach [the call sites you mentioned](http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp/Binder/Semantics/OverloadResolution/OverloadResolution.cs,1674). I've tried adding `throw new Exception()` and running all the tests and they all continue to pass.